src/labels/CustomLabels.labels

```diff
@@ -432,6 +432,14 @@ day:      &apos;day&apos;</value>
         <value>Copy Schedule From</value>
     </labels>
     <labels>
+        <fullName>labelEventInThePast</fullName>
+        <categories>VolunteersJobListingFS</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>labelEventInThePast</shortDescription>
+        <value>This event is not accepting signups at this time.</value>
```

Comment:
Please change this to "This job has no upcoming shifts."

V4S doesn't use the terminology Event.  I realize UNH wants it to be event, but they can certainly translate the label.